apps/rush-lib/src/logic/PackageChangeAnalyzer.ts

```diff
@@ -56,10 +59,32 @@ export class PackageChangeAnalyzer {
     let repoDeps: IPackageDeps;
     try {
       if (this._isGitSupported) {
+        const commandLineConfigFile: string = path.join(
+          this._rushConfiguration.commonRushConfigFolder,
+          RushConstants.commandLineFilename
+        );
+
+        const commandLineConfiguration: CommandLineConfiguration = CommandLineConfiguration.loadFromFileOrDefault(
+          commandLineConfigFile
+        );
+
+        // Generate possible package-deps filenames from commands in order to
+        // ensure they are not included in the repoDeps
+        const excludes: string[] = commandLineConfiguration.commands.reduce(
+          (filenames: string[], command: CommandJson) => {
+            if (command.commandKind === 'bulk') {
+              filenames.push(Utilities.getPackageDepsFilenameForCommand(command.name));
+            }
+
+            return filenames;
+          },
+          []
+        );
+
         // Load the package deps hash for the whole repository
         repoDeps = PackageChangeAnalyzer.getPackageDeps(
           this._rushConfiguration.rushJsonFolder,
-          [RushConstants.packageDepsFilename]
+          excludes
```

Comment:
It looks like this parameter has never actually worked correctly, so I'm going to just remove this.